### ISSUE TITLE
Reimplement tests for uninitalized temporaries

### ISSUE THREAD

**Issue by twicki** [MeteoSwiss-APN/dawn#339]
This test was very basic and removed with [this](https://github.com/MeteoSwiss-APN/gtclang/pull/127) PR upon disabling the (broken) pass

**Comment by Stagno** [comment]
@twicki still open, right?

**Comment by twicki** [comment]
yes, the pass is currently disabled , see here: https://github.com/MeteoSwiss-APN/dawn/blob/7ab314894dc45e60284d509626f1a3bbfa52739f/dawn/src/dawn/Compiler/DawnCompiler.cpp#L185
